homeassistant/components/telegram_bot/__init__.py

```diff
@@ -135,7 +135,7 @@
         vol.Optional(ATTR_DISABLE_WEB_PREV): cv.boolean,
         vol.Optional(ATTR_KEYBOARD): vol.All(cv.ensure_list, [cv.string]),
         vol.Optional(ATTR_KEYBOARD_INLINE): cv.ensure_list,
-        vol.Optional(CONF_TIMEOUT): vol.Coerce(float),
+        vol.Optional(ATTR_TIMEOUT): cv.positive_int,
```

Comment:
@MartinHjelmare does changing the expected data type of a service count as a breaking change?

Reply:
It could but here I think we're backwards compatible. We're limiting it to positive numbers though. But a negative timeout sounds weird so I think that's ok.